src/plugins/intel_gna/tests/unit/gna_scratch_area_size_test.cpp

```diff
@@ -0,0 +1,296 @@
+                                  GetParam().scratch_region_size) {}
+
+private:
+    std::shared_ptr<ngraph::Function> create_model() override {
+        ConvolutionTestParams param = GetParam();
+        const ov::element::Type ng_prec = ngraph::element::f32;
+        const auto input_shape = std::make_shared<ngraph::opset9::Parameter>(ng_prec, param.input_shape);
+        std::shared_ptr<ngraph::Node> maxpool;
+        std::shared_ptr<ngraph::Node> activation;
+        std::shared_ptr<ngraph::Node> result_operation;
+
+        result_operation = ngraph::builder::makeConvolution(
+            input_shape,
+            ng_prec,
+            param.kernels_shape,
+            // Set the stride and kernel shape to the same value in order to force padding equal to 0
+            param.kernels_shape,
+            {0, 0},
+            {0, 0},
+            {1, 1},
+            ngraph::op::PadType::VALID,
+            param.channels);
+
+        if (param.maxpool_kernels_shape.size()) {
+            maxpool = ngraph::builder::makePooling(
+                result_operation,
+                // Set the stride and kernel shape to the same value in order to force padding equal to 0
+                param.maxpool_kernels_shape,
+                {0, 0},
+                {0, 0},
+                param.maxpool_kernels_shape,
+                ngraph::op::RoundingType::FLOOR,
+                ngraph::op::PadType::VALID,
+                false,
+                PoolingTypes::MAX);
+            result_operation = maxpool;
+        }
+
+        if (param.activation != ActivationTypes::None) {
+            activation = ngraph::builder::makeActivation(result_operation, ng_prec, param.activation);
+            result_operation = activation;
+        }
+
+        auto result = std::make_shared<ngraph::opset9::Result>(result_operation);
+
+        return std::make_shared<ngraph::Function>(ngraph::ResultVector{result},
+                                                  ov::ParameterVector{input_shape},
+                                                  "Convolution");
+    }
+};
+
+struct FullyConnectedTestParams {
+    DeviceVersion target;
+    size_t scratch_region_requests_cnt;
+    size_t scratch_region_size;
+    ngraph::Shape input_shape;
+    size_t out_size;
+    ActivationTypes activation;
+};
+
+class FullyConnectedScratchAreaSizeTest : public ScratchAreaSizeTestBase<FullyConnectedTestParams> {
+public:
+    FullyConnectedScratchAreaSizeTest()
+        : ScratchAreaSizeTestBase(GetParam().target,
+                                  GetParam().scratch_region_requests_cnt,
+                                  GetParam().scratch_region_size) {}
+
+private:
+    std::shared_ptr<ngraph::Function> create_model() override {
+        FullyConnectedTestParams param = GetParam();
+        const ov::element::Type ng_prec = ngraph::element::f32;
+        const auto input_shape = std::make_shared<ngraph::opset9::Parameter>(ng_prec, param.input_shape);
```

Comment:
Please use opset11 and consider `using namespace ngraph::opset11`. Also, check other places